modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/controls/BulkActions.js

```diff
@@ -106,6 +108,80 @@ function BulkActions({
 		}
 	}
 
+	const renderNewBulkActions = ({
+		formId,
+		formName,
+		loadData,
+		namespace,
+		sidePanelId,
+	}) => {
+		return (
+			<ul className="bulk-actions navbar-nav">
+				{bulkActions.map((actionDefinition, i) => (
+					<li className="nav-item" key={actionDefinition.label}>
```

Comment:
Is there a Clay component with this `ul.navbar-nav > li.nav-item` structure? If not, this is fine.

Reply:
Ya, I can use [Nav](https://clayui.com/docs/components/nav.html) but since this is just adding just the button I decided to not use the Nav component for this and then in [this next ticket that updates the other action buttons](https://liferay.atlassian.net/browse/LPD-42584) I'll update the entire structure to make use of the Nav clay component.